lib/kms.js

```diff
@@ -37,12 +63,22 @@ export async function createKeystore({
     keystoreConfig.referenceId = referenceId;
   }
   const {httpsAgent} = brHttpsAgent;
-  return KmsClient.createKeystore({
-    url: `${bedrock.config.tokenizer.kms.baseUrl}/keystores`,
-    config: keystoreConfig,
-    invocationSigner,
-    httpsAgent
-  });
+  let result;
+  try {
+    result = await KmsClient.createKeystore({
+      url: `${bedrock.config.tokenizer.kms.baseUrl}/keystores`,
+      config: keystoreConfig,
+      invocationSigner,
+      httpsAgent
+    });
+  } catch(cause) {
+    const error = new BedrockError(
```

Comment:
This feels like it's a layering violation. Can we move it to where we call `_createTokenizer` instead? Then we can just wrap the call of that function once and generate an error there with any cause that was thrown just once.

Reply:
K, will move.